docs/visual-basic/programming-guide/concepts/linq/how-to-join-content-from-dissimilar-files-linq.md

```diff
@@ -95,25 +100,30 @@ Class JoinStrings
     End Sub  
 End Class  
 ' Output:  
-'Merge two spreadsheets:  
-'Adams,Terry,120, 99, 82, 81  
-'Fakhouri,Fadi,116, 99, 86, 90  
-'Feng,Hanying,117, 93, 92, 80  
-'Garcia,Cesar,114, 97, 89, 85  
-'Garcia,Debra,115, 35, 72, 91  
-'Garcia,Hugo,118, 92, 90, 83  
-'Mortensen,Sven,113, 88, 94, 65  
-'O'Donnell,Claire,112, 75, 84, 91  
-'Omelchenko,Svetlana,111, 97, 92, 81  
-'Tucker,Lance,119, 68, 79, 88  
-'Tucker,Michael,122, 94, 92, 91  
-'Zabokritski,Eugene,121, 96, 85, 91  
-'12 total names in list  
+' Merge two spreadsheets:
+' Omelchenko, 97, 92, 81, 60
+' O'Donnell, 75, 84, 91, 39
+' Mortensen, 88, 94, 65, 91
+' Garcia, 97, 89, 85, 82
+' Garcia, 35, 72, 91, 70
+' Fakhouri, 99, 86, 90, 94
+' Feng, 93, 92, 80, 87
+' Garcia, 92, 90, 83, 78
+' Tucker, 68, 79, 88, 92
+' Adams, 99, 82, 81, 79
+' Zabokritski, 96, 85, 91, 60
+' Tucker, 94, 92, 91, 91
+' 12 total names in list 
 ```  
-  
-## Compiling the Code  
- Create a project that targets the .NET Framework version 3.5 or higher with a reference to System.Core.dll and a `Imports` statement for the System.Linq namespace.  
-  
-## See Also  
+
+## Compiling the code
+
+Create and compile a project that targets one of the following options:
+
+- .NET Framework version 3.5 or higher with a reference to System.Core.dll.
```

Comment:
A reference to System.Core isn't needed when compiling starting with (I think) 4.0. I think this might be best as two separate lines, one for .NET Framework 3.5 (with System.Core.dll) and the second for 4.0 or later.

Reply:
@rpetrusha is this remark also valid for the C# topic?

Reply:
Sorry, @pkulikov. I should have commented on both topics, not just the VB one. Yes, it is.